### ISSUE TITLE
Gazebo_worlds in kinetic

### ISSUE THREAD

**Issue by Plzr** [a-price/reactor_controller#2]
Hi, 

It seems like gazebo_worlds is no longer supported in kinetic. When I try to roslaunch this package, I get the following reply. Any ideas?
```

remy-xubuntu@remy-xubuntu:~/catkin_a_price$ roslaunch reactor_description display.launch
... logging to /home/remy-xubuntu/.ros/log/6942a260-b716-11e6-97cb-0024d729e72c/roslaunch-remy-xubuntu-14403.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://remy-xubuntu:43142/

SUMMARY
========

PARAMETERS
 * /robot_description: <...>
 * /rosdistro: kinetic
 * /rosversion: 1.12.2
 * /use_gui: False

NODES
  /
    joint_state_publisher (joint_state_publisher/joint_state_publisher)
    robot_state_publisher (robot_state_publisher/state_publisher)
    rviz (rviz/rviz)

auto-starting new master
process[master]: started with pid [14414]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to 6942a260-b716-11e6-97cb-0024d729e72c
process[rosout-1]: started with pid [14427]
started core service [/rosout]
process[joint_state_publisher-2]: started with pid [14438]
process[robot_state_publisher-3]: started with pid [14442]
process[rviz-4]: started with pid [14446]
[rviz-4] process has finished cleanly
log file: /home/remy-xubuntu/.ros/log/6942a260-b716-11e6-97cb-0024d729e72c/rviz-4*.log
^C[robot_state_publisher-3] killing on exit
[joint_state_publisher-2] killing on exit
[rosout-1] killing on exit
[master] killing on exit
shutting down processing monitor...
... shutting down processing monitor complete
done
remy-xubuntu@remy-xubuntu:~/catkin_a_price$ roslaunch reactor_description gazebo.launch
... logging to /home/remy-xubuntu/.ros/log/72d207d0-b716-11e6-97cb-0024d729e72c/roslaunch-remy-xubuntu-14603.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/__init__.py", line 307, in main
    p.start()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/parent.py", line 268, in start
    self._start_infrastructure()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/parent.py", line 217, in _start_infrastructure
    self._load_config()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/parent.py", line 132, in _load_config
    roslaunch_strs=self.roslaunch_strs, verbose=self.verbose)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/config.py", line 451, in load_config_default
    loader.load(f, config, verbose=verbose)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 746, in load
    self._load_launch(launch, ros_config, is_core=core, filename=filename, argv=argv, verbose=verbose)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 718, in _load_launch
    self._recurse_load(ros_config, launch.childNodes, self.root_context, None, is_core, verbose)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 682, in _recurse_load
    val = self._include_tag(tag, context, ros_config, default_machine, is_core, verbose)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 95, in call
    return f(*args, **kwds)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 587, in _include_tag
    inc_filename = self.resolve_args(tag.attributes['file'].value, context)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/xmlloader.py", line 183, in resolve_args
    return substitution_args.resolve_args(args, context=context.resolve_dict, resolve_anon=self.resolve_anon)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 366, in resolve_args
    resolved = _resolve_args(resolved, context, resolve_anon, commands)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 379, in _resolve_args
    resolved = commands[command](resolved, a, args, context)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 150, in _find
    source_path_to_packages=source_path_to_packages)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/roslaunch/substitution_args.py", line 193, in _find_executable
    full_path = _get_executable_path(rp.get_path(args[0]), path)
  File "/usr/lib/python2.7/dist-packages/rospkg/rospack.py", line 200, in get_path
    raise ResourceNotFound(name, ros_paths=self._ros_paths)
ResourceNotFound: gazebo_worlds
ROS path [0]=/opt/ros/kinetic/share/ros
ROS path [1]=/home/remy-xubuntu/catkin_a_price/src
ROS path [2]=/opt/ros/kinetic/share
ROS path [3]=/opt/ros/kinetic/stacks

```

**Comment by a-price** [comment]
Hi @Plzr,

Try the gazebo branch at https://github.com/a-price/reactor_description/tree/gazebo. We created it a month ago, but I've been waiting on confirmation it works from someone before pulling it into master. If it works for you I'll merge it in.

**Comment by Plzr** [comment]
Hey,

Your package seems to work: Gazebo launches without an error, and the model is visible. As I don't know yet how to control it, I can not say anything about the ROS connection to gazebo though. 

Thanks for helping!

**Comment by a-price** [comment]
Great, I'm marking as resolved by 505630314ec59456fa02bdcc933e35b5adaaafb2.
